zero:
  ID: 3
  Operating_Mode: 0
  Current_Limit: 50
  Torque_Enable: 1